packages/authgear-react-native/index.d.ts

```diff
@@ -928,56 +933,94 @@ export declare class ReactNativeContainer {
    */
   configure(options: ConfigureOptions): Promise<void>;
   /**
-   * Authenticate the end user via the web.
+   * authenticate() starts the authentication process in a webview.

```

Comment:
Change the term `webview` to `@link UIImplementation`. These methods are actually do this.

Reply:
ok